ui/i18n/src/main/res/values/strings.xml

```diff
@@ -18,6 +18,7 @@
     <string name="downloads_label">Downloads</string>
     <string name="open_autodownload_settings">Open settings</string>
     <string name="downloads_log_label">Download log</string>
+    <string name="delete_downloads_played">Delete played downloads</string>
```

Comment:
Maybe just "Delete played" to keep the title short? If it is only shown on the downloads screen, it should be clear that it is about downloads. Especially when translated, the full text would make the menu entry super long